master/internal/experiment.go

```diff
@@ -832,7 +835,6 @@ func (e *internalExperiment) processOperations(
 			if err := e.searcher.SetCustomSearcherProgress(op.Progress); err != nil {
 				e.syslog.WithError(err).Error("failed to set searcher progress")
 			}
-
```

Comment:
nit: is this intentional?

Reply:
yeah, there's a weird blank line between switch cases